daemon/mgr/container.go

```diff
@@ -182,7 +179,7 @@ func NewContainerManager(ctx context.Context, store *meta.Store, cli ctrd.APICli
 
 	go mgr.execProcessGC()
 
-	return mgr, nil
+	return mgr, mgr.Restore(ctx)
```

Comment:
In the intialization of container manager, I suggest we should not add restore action here. It will break the pure functionality of function `NewContainerManager `.
As for adding this feature, I am wondering if we could add this function in 
```
	containerMgr, err := internal.GenContainerMgr(ctx, d)
	if err != nil {
		return err
	}
	d.containerMgr = containerMgr
```

As a result, the code will be like
```
	containerMgr, err := internal.GenContainerMgr(ctx, d)
	if err != nil {
		return err
	}
	d.containerMgr = containerMgr
    if err := containerMgr.Restore(); err != nil {
        return err
    }
```

Reply:
fine!